lib/functionTemplates/js/binaris.d.ts

```diff
@@ -0,0 +1,28 @@
+/// <reference types="node" />
+interface BinarisHTTPRequest {
+  env: Record<string, string | undefined>;
+  query: Record<string, string | string[] | undefined>;
+  body: Buffer;
+  path: string;
+  method: string;
+  requestId: string;
+  headers: Record<string, string | string[] | undefined>;
+}
+
+interface FunctionResponse {
+  statusCode: number;
+  headers: Record<string, string | undefined>;
+  body: Buffer | string;
+}
+
+declare class BinarisHTTPResponse {
+  userResponse: Partial<FunctionResponse>;
+  constructor(userResponse: Partial<FunctionResponse>);
+}
+
+declare type FunctionContext = {
+  request: BinarisHTTPRequest;
+  HTTPResponse: typeof BinarisHTTPResponse;
+};
+
+export declare type BinarisFunction = (body: unknown, context: FunctionContext) => Promise<any>
```

Comment:
Don't we know the generic form of the body?

Reply:
It can be whatever you pass in the body...